knowledge-base/create-pdf-document-with-logo-and-text-using-fixedcontenteditor.md

```diff
@@ -0,0 +1,175 @@
+---
+title: How to Generate a PDF Document with Logo and Text using FixedContentEditor 
+description: Learn how to create a PDF document from scratch containing a logo and text content using the FixedContentEditor of RadPdfProcessing.
+type: how-to
+page_title: How to Generate a PDF Document with Logo and Text using FixedContentEditor 
+slug: create-pdf-document-with-logo-and-text-using-fixedcontenteditor
+tags: pdf, fixedcontenteditor, editor, logo, image, text, processing
+res_type: kb
+---
+# Environment
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 2024.1.124 | RadPdfProcessing |[Desislava Yordanova](https://www.telerik.com/blogs/author/desislava-yordanova)| 
+
+# Description
+
+This tutorial demonstrates a sample approach how to create a PDF document from scratch that contains a logo and text. 
+
+# Solution
+
+The powerful [FixedContentEditor]({%slug radpdfprocessing-editing-fixedcontenteditor%}) allows you draw any element at the desired [Position]({%slug radpdfprocessing-concepts-position%}):
+
+```
+        public static System.Windows.Size pageSize = new System.Windows.Size(Unit.MmToDip(210), Telerik.Windows.Documents.Media.Unit.MmToDip(297));
+        public static Padding pageMarginsValue = new Telerik.Windows.Documents.Primitives.Padding(
+             Unit.MmToDip(20),//left
+              Unit.MmToDip(0),//top
+               Unit.MmToDip(0),//right
+                Unit.MmToDip(0));//bottom
+        public static double currentYposition = pageMarginsValue.Top;
+        public static double currentXposition = pageMarginsValue.Left;
+        public static double pageYLimit = pageSize.Height - pageMarginsValue.Bottom;
+        public static double pageXLimit = pageSize.Width - pageMarginsValue.Right;
+        static void Main(string[] args)
+        {
+            RadFixedDocument fixedDocument = new RadFixedDocument();
+
+            DrawLogo(fixedDocument);
+            DrawHeading(fixedDocument);
+            DrawText(fixedDocument);
+
+
+            PdfFormatProvider provider = new PdfFormatProvider();
+            string outputFilePath = @"..\..\sample.pdf";
+            File.Delete(outputFilePath);
+            using (Stream output = File.OpenWrite(outputFilePath))
+            {
+                provider.Export(fixedDocument, output);
+            }
+            Process.Start(outputFilePath);
```

Comment:
This might not work in every environment, Maybe use 
` 
var psi = new ProcessStartInfo()
{
FileName = saveFile,
 UseShellExecute = true
 };
Process.Start(psi);
`